salt/master.py

```diff
@@ -768,7 +768,7 @@ def __setup_fileserver(self):
         '''
         Set the local file objects from the file server interface
         '''
-        fs_ = salt.fileserver.Fileserver(self.opts)
+        self.fs_ = fs_ = salt.fileserver.Fileserver(self.opts)
```

Comment:
Can this just be:
self.fs_ = salt.fileserver.Fileserver(self.opts)
and then change the fs refs below?